app/src/main/cpp/native-lib.cpp

```diff
@@ -188,6 +188,15 @@ android_main(android_app *aAppState) {
   aAppState->userData = sAppContext.get();
   aAppState->onAppCmd = CommandCallback;
 
+#ifdef PICOOXR
+  if (!sAppContext->mEgl && aAppState->window) {
+    // Work around APP_CMD_INIT_WINDOW and APP_CMD_RESUME callback not
+    // firing on Pico 4 when starting the app
```

Comment:
This is interesting, is there any bug open in a public repository about this. If that's the case we should include a link here.

Reply:
I found this issue while debugging and only saw it in Wolvic, so I don't think there is a mention anywhere. It is also unlikely to be a general issue, as I did not see this with the Godot game engine. I'd guess that the callback is not yet registered when the init and resume events happen, possibly during the setup of Pico's VR runtime bits that "upgrade" a normal android activity. Whether or not this is caused by Wolvic or Pico is not something I can say for certain, though.